ui/core/components/inputs/consumables.ts

```diff
@@ -1019,10 +1019,20 @@ export const WildStrikes: ConsumableInputConfig<WeaponImbue> = {
 // Other Imbues
 
 // Wizard Oils
-export const BrillianWizardOil = (slot: ItemSlot): ConsumableInputConfig<WeaponImbue> => {
+export const EnchantedRepellent = (slot: ItemSlot): ConsumableInputConfig<WeaponImbue> => {
+	return {
+		actionId: player => player.getMatchingItemActionId([{ id: 233996, minLevel: 60 }]),
+		value: WeaponImbue.EnchantedRepellent,
+		showWhen: player => {
+			const weapon = player.getEquippedItem(slot);
+			return !weapon || isWeapon(weapon.item.weaponType);
```

Comment:
I think this won't allow it to be put on a shield, but this particular imbue can be put on both weapons and shields. You'd want to make sure it's either a weapon or a shield, or just say `!weapon || weapon.item.weaponType != WeaponType.WeaponTypeOffhand`